src/mapping/parser/grammar.pest

```diff
@@ -42,37 +42,15 @@ path_coalesce = !{ "(" ~ (path_segment | quoted_path_segment) ~ ("|" ~ (path_seg
 
 dot_path = ${ ("." ~ (path_segment | quoted_path_segment | path_coalesce))+ }
 
-// Query functions
-//
-// NOTE: Not including delete here yet, but there's no reason we can't slot it
-// in at some point and get `.nested = del(.foo)`, same with `only_fields`.
-query_function = {
-    to_string |
-    to_int |
-    to_float |
-    to_bool |
-    to_timestamp |
-    parse_timestamp |
-    upcase |
-    downcase |
-    uuid_v4 |
-    sha1 |
-    md5 |
-    now
-}
+ident = @{ ASCII_ALPHA_LOWER ~ ( ASCII_ALPHA_LOWER | "_" )* }
```

Comment:
Traditionally, digits are allowed in identifiers as is leading underscore, though we don't use either (yet?).

Reply:
We actually do use digits in `md5` etc. I've updated the rule to accept digits and uppercase characters (but not leading underscores, for now).